### PR TITLE
TF-816: get list email forward

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/185285736-8d406f3a-ed18-4cd2-854d-a3662ecac79a.mov

https://user-images.githubusercontent.com/107173849/185285698-d47f6552-550e-4e3d-8803-4f2099a0f098.mov

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: lib/features/manage_account/data/network/manage_account_api.dart [hunk 1]
```diff
@@ -180,4 +185,31 @@ class ManageAccountAPI {
...
+    final processingInvocation = ProcessingInvocation();
+    final requestBuilder = JmapRequestBuilder(_httpClient, processingInvocation);
+
+    final getForwardMethod = GetForwardMethod(
+      accountId,
+    )..addIds({ForwardIdSingleton.forwardIdSingleton.id});
+
+    final getForwardInvocation = requestBuilder.invocation(getForwardMethod);
+    final response = await (requestBuilder
+      ..usings(getForwardMethod.requiredCapabilities))
```
Comment (dab246) [inline lib/features/manage_account/data/network/manage_account_api.dart:199]: add indent

File: lib/features/manage_account/data/repository/manage_account_repository_impl.dart [hunk 2]
```diff
@@ -63,4 +64,10 @@ class ManageAccountRepositoryImpl extends ManageAccountRepository {
   Future<List<TMailRule>> editEmailRuleFilter(AccountId accountId, EditEmailRuleFilterRequest ruleFilterRequest) {
     return dataSource.editEmailRuleFilter(accountId, ruleFilterRequest);
   }
+
+  @override
+  Future<TMailForward> getForward(AccountId accountId) {
+    return dataSource.getForward(accountId);
+
```
Comment (dab246) [inline lib/features/manage_account/data/repository/manage_account_repository_impl.dart:71]: remove excess line

File: lib/features/manage_account/presentation/forward/forward_controller.dart [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  @override
+  void onError(error) {}
+
+  @override
+  void onInit() {
+    _getForward();
+    super.onInit();
+  }
+
```
Comment (dab246) [inline lib/features/manage_account/presentation/forward/forward_controller.dart:38]: remove excess line

File: lib/features/manage_account/presentation/forward/forward_controller.dart [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (dab246) [inline lib/features/manage_account/presentation/forward/forward_controller.dart:39]: same

File: lib/features/manage_account/presentation/forward/forward_view.dart [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        return const EdgeInsets.only(right: 32, top: 16, bottom: 16);
+      } else {
+        return const EdgeInsets.only(top: 16, bottom: 16, right: 28, left: 20);
+      }
+    }
+  }
+
+  Widget _buildLoadingView() {
+    return Obx(() => controller.viewState.value.fold(
+            (failure) => const SizedBox.shrink(),
```
Comment (dab246) [inline lib/features/manage_account/presentation/forward/forward_view.dart:58]: remove indent

File: lib/features/manage_account/presentation/forward/forward_view.dart [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            (success) => success is LoadingState
```
Comment (dab246) [inline lib/features/manage_account/presentation/forward/forward_view.dart:59]: same

**Review by dab246 — APPROVED** [review]

**Pull request merged**